.github/linters/.hadolint.yaml

```diff
@@ -14,4 +14,4 @@
 ---
 ignored:
   - DL3008 # allow non-pinned versions
-  - DL3004 # using sudo
+  - DL3013 # pip packages without requirements.txt
```

Comment:
any ideas how to fix this... ?

Reply:
Well, I e.g. don't mind the pinned versions as it gives me a reference on when/how it was working on another machine even if it may not work for me 🤷‍♀️

Reply:
The non-serious answer would be: choose a different linter 🤣

Reply:
> Well, I e.g. don't mind the pinned versions as it gives me a reference on when/how it was working on another machine even if it may not work for me 🤷‍♀️

Sorry, was refering to the diff in this PR

Reply:
Yeah, I thought that this is what you meant and I was just reading up about this as it seems that it's very common for hadolint users to ignore this error (which in my mind defeats the point). I did find some suggestions on that there was a fix somewhere, but didn't get to the details just yet ...

Reply:
I was trying to make a different point though, see slack 😉